examples/step-22/step-22.cc

```diff
@@ -216,37 +218,40 @@ namespace Step22
 
 
   // We implement similar functions for the right hand side which for the
-  // current example is simply zero:
+  // current example is simply zero. Since we have the incompressibility
+  // constraint, the dim-th component of the right hand side will always be 0
+  // and therefore we return a dim dimensional Tensor which represents
+  // only the external forces $\textbf f$.
```

Comment:
Let's make this as follows, omitting the details of the ordering and instead talking about what the function represents (which you already do in some sense):
```suggestion
  // current example is simply zero. In the current case, what we want to represent is the
  // right hand side of the velocity equation, that is, the body forces acting on the
  // fluid. (The right hand side of the divergence equation
  // is always zero, since we are dealing with an incompressible medium.) As such, the
  // right hand side is represented by a `Tensor<1,dim>` object: a dim-dimensional vector,
  // and the right tool to implement such a function is to derive it from `TensorFunction<1,dim>`:
```